src/pheval/analyse/generate_summary_outputs.py

```diff
@@ -118,8 +100,8 @@ def generate_benchmark_gene_output(
     prioritisation_data: TrackRunPrioritisation, plot_type: str
 ) -> None:
     """Generate gene prioritisation outputs for benchmarking single run."""
-    RankComparisonGenerator(prioritisation_data.gene_prioritisation.ranks).generate_gene_output(
-        f"{prioritisation_data.gene_prioritisation.results_dir.name}"
+    RankComparisonGenerator(prioritisation_data.gene_prioritisation.ranks).generate_output(
+        f"{prioritisation_data.gene_prioritisation.results_dir.name}", "-gene_rank_comparison.tsv"
```

Comment:
Technically this filename seems to violate SRP (because it exists more than once in the code).

Reply:
The construction of the full filename? Or the `"-gene_rank_comparison.tsv"` part?

Reply:
The string "-gene_rank_comparison.tsv" appears twice in that file.

Reply:
Ah yes! I plan on addressing this in #193. I have the code roughly written out with how I plan it to be structured and it avoids this